### ISSUE TITLE
Make 0.setup.R agile to config.yml

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/CostelloPSCNSeq#18]
Creating a new issue based on the below:

@ivan108 [wrote](https://github.com/HenrikBengtsson/Costello-PSCN-Seq/pull/17#issuecomment-444739637):

> I wonder what does it take to move that path into config.yml?

@henrikbengtsson [replied](https://github.com/HenrikBengtsson/Costello-PSCN-Seq/pull/17#issuecomment-444741240):

> One easier, quicker approach is to be able to override the default via an env var. See ?Sys.getenv.

**Comment by HenrikBengtsson** [comment]
So, I completely forgot that there already was `config.yml` file in this project; I read your initial comment as a request to add a `config.yml` file.

After re-reading, I now take it as you wanted `0.setup.R` to be agile to whatever is in the `config.yml` file.  Maybe we could add support for that.  

If I recall it correctly, the main purpose of the `0.setup.R` file was to programmatically "communicate" what the intent of the different folders is instead of trying to cover it all in the documentation.  I'll add it to the wish list.

**Comment by HenrikBengtsson** [comment]
@ivan108 [wrote](https://github.com/HenrikBengtsson/Costello-PSCN-Seq/pull/17#issuecomment-445043637):
> Quick question, in default config.yml file the "dataset" is defined as "CostelloP_2015-Exome,bwa,realigned,rmDups,recal" and in 0.setup.R the "path" is defined as "bamData/CostelloP_2015-Exome,bwa,realigned,rmDups,recal/". So if I change dataset name, shall I also change "path" or it doesn't matter?

Yes, it makes sense to at least keep `config.yml` and `0.setup.R` consistent this way.

PS. Maybe this adds another argument for making `0.setup.R` follow whatever is set in `config.yml`.

**Comment by ivan108** [comment]
Yes, that was my original thought, since we already have config.yml, why not to use it in the 0.setup.R, instead of hard coding...
